src/screens/SignUp/index.tsx

```diff
@@ -77,12 +83,27 @@ function SignUp() {
           errorMessage={errors.confirmPassword?.message}
         />
         {error && (
-          <p className={styles.error}>{i18next.t(`${I18N_CONFIG.key}:${ERROR_MESSAGES.signUpService}`)}</p>
+          <span role="error" className={styles.error}>
+            {i18next.t(`${I18N_CONFIG.key}:${ERROR_MESSAGES.signUpService}`)}
+          </span>
         )}
-        <button className={`${styles.submitButton} ${styles.formButton} m-bottom-5 full-width`} type="submit">
+        {state && (
```

Comment:
I'm not sure if this is the best way to handle successful response. Better way for me it's using withPostSuccess function and setting a success state and checking it